mat64/matrix.go

```diff
@@ -295,11 +295,8 @@ func Cond(a Matrix, norm float64) float64 {
 	case 1:
 		lnorm = lapack.MaxColumnSum
 	case 2:
-		// Use SVD to compute the condition number.
-		// TODO(btracey): Replace this with temporary workspace when SVD is fixed.
-		tmp := NewDense(m, n, nil)
-		tmp.Copy(a)
-		svd := SVD(tmp, math.Pow(2, -52.0), math.Pow(2, -966.0), false, false)
+		var svd SVD
+		svd.Factorize(a, matrix.SVDNone)
```

Comment:
I'm having a tough time seeing if the decomposition only fails when the input matrix is singular or near-singular. I want to say that if it fails, we can return math.Inf(1).